### ISSUE TITLE
it dosnt start what do i do

### ISSUE THREAD

**Issue by THEman6989** [Pikurrot/whisper-gui#1]
Activating WhisperX conda environment...
Running WhisperX...
Traceback (most recent call last):
  File "C:\Users\Stabl\whisper-gui\main.py", line 1, in <module>
    import gradio as gr
ModuleNotFoundError: No module named 'gradio'
Drücken Sie eine beliebige Taste . . .

**Comment by Pikurrot** [comment]
Hi, you need to install the `gradio` library first inside your environment. You can do it by running these commands in your terminal:

`conda activate whisperx`
`pip install gradio==3.23.0`

Then you should be able to run the program.

Thanks for pointing that out, I will update the instructions. Let me know if you have any more problems.

**Comment by THEman6989** [comment]
Thanks

**Comment by THEman6989** [comment]
Are you going to update the ui so that large v3 is selectable

**Comment by Pikurrot** [comment]
Yes, I will try that the UI accepts large-v3, and further on custom downloaded models, that are not WhisperX. I can add the beam size as an adjustable parameter as well.

**Comment by THEman6989** [comment]
cool and what is about Speaker Diarization

**Comment by Pikurrot** [comment]
I will check if that's a possibility, it would be great.
Also, if you like this project, I invite you to contribute to it and make the changes you like.

**Comment by THEman6989** [comment]
I would really like doing it but i am really new to this space and cant code python.
do you know where i can learn to code with python in the area of ai

**Comment by Pikurrot** [comment]
I suggest learning python at Codecademy (won't take too much time), then it's a matter of learning and mastering its libraries for AI: numpy, pandas, scipy, scikit-learn... and later pytorch and tensorflow. You can search any course that expands python on the area of AI using these libraries, but looking at their docs is also helpful.
Personally, I also learned a lot with chatgpt :)
